### PR TITLE
[Configs] Remove unnecessary dependency.

### PR DESCRIPTION
**Pull request by JoshLind**
### Description

This PR removes an unnecessary dependency on `PersistableConfig` from the rosetta smoke tests.

### Test Plan
Existing test infrastructure.

### CODE REVIEW

File: testsuite/smoke-test/src/rosetta.rs [hunk 1]
```diff
@@ -1339,8 +1336,10 @@ async fn parse_operations(
                         native_coin(),
                         "Balance should be the native coin"
                     );
-                    let delta =
-                        u64::parse(&amount.value).expect("Should be able to parse amount value");
+                    let delta = amount
+                        .value
+                        .parse::<u64>()
+                        .expect("Should be able to parse amount value");
```
Comment (gregnazario) [inline testsuite/smoke-test/src/rosetta.rs:1342]: FYI changes like this should probably be in a separate commit.

**Review by gregnazario — APPROVED** [review]

File: testsuite/smoke-test/src/rosetta.rs (hunk 1 above)
Comment (JoshLind) [reply testsuite/smoke-test/src/rosetta.rs:1342]: These are the changes 😆 The rosetta smoke tests incorrectly relied on the `parse` method from the `PersistentConfig`, so I've cut that dependency and updated the parsing logic 😄

**Review by moonclavedev — APPROVED** [review]
